src/constants/defaultReplies.js

```diff
@@ -0,0 +1,20 @@
+export const errorResponses = {
```

Comment:
fix for [#332](https://github.com/webaverse-studios/CharacterCreator/issues/332)